.github/workflows/release.yml

```diff
@@ -0,0 +1,25 @@
+name: release
+
+on:
+  workflow_dispatch:
+    inputs:
+      semver:
+        description: "The semver to use"
+        required: true
+        default: "patch"
+        type: choice
+        options:
+          - patch
+          - minor
+          - major
+  pull_request:
+    types: [closed]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: nearform/optic-release-automation@main
```

Comment:
use the last major tag of the action instead of @main please. i.e. @v4

Reply:
Done.